modules/gateways/blockonomics/blockonomics.php

```diff
@@ -521,6 +521,12 @@ public function processOrderHash($order_hash, $blockonomics_currency)
         }
         // Process a new order for the id and blockonomics currency
         $new_order = $this->createNewCryptoOrder($order_info, $blockonomics_currency);
+        $log_data = array(
+            'invoice_id' => $new_order->id_order,
+            'address' => $new_order->addr,
+            'crypto' => $new_order->blockonomics_currency
+        );
+        logTransaction('blockonomics', $log_data, 'New Order Created');
```

Comment:
Instead of 'blockonomics' as the module name, we should collect it from the gateway variables as in the callback logs:
```
$gatewayModuleName = 'blockonomics';
// Fetch gateway configuration parameters.
$gatewayParams = getGatewayVariables($gatewayModuleName);
```
 and use as follows:
```
$gatewayParams['name']
```